app/containers/Header/HeaderForm.js

```diff
@@ -5,106 +5,146 @@
-    />
-  );
+export const LoginProfile = React.memo(
+  ({
+    profileInfo,
+    showSignUpModalDispatch,
+    showLoginModalDispatch,
+    isMenuVisible,
+    expandLeftMenuNavigation,
+  }) => {
+    if (profileInfo) {
+      return (
+        <UserProfileNav
+          isMenuVisible={isMenuVisible}
+          profileInfo={profileInfo}
+          expandLeftMenuNavigation={expandLeftMenuNavigation}
+        />
+      );
+    }
+
+    return (
+      <UserAuthNavLinks
+        isMenuVisible={isMenuVisible}
+        showSignUpModal={showSignUpModalDispatch}
+        showLoginModal={showLoginModalDispatch}
+      />
+    );
+  },
+);
+
+const homeRoute = routes.home();
+const addQuestionRoute = () => {
+  createdHistory.push(routes.question_ask());
 };
 
-/* eslint-disable react/prefer-stateless-function */
 const HeaderForm = props => (
-  <Wrapper>
-    <nav className="container navbar navbar-expand-lg navbar-light">
-      <Link to={routes.home()} href={routes.home()}>
-        <Logo />
-      </Link>
-
-      <button
-        className="navbar-toggler"
-        type="button"
-        data-toggle="collapse"
-        data-target="#navbarTogglerDemo02"
-        aria-controls="navbarTogglerDemo02"
-        aria-expanded="false"
-        aria-label="Toggle navigation"
-      >
-        <span className="navbar-toggler-icon" />
-      </button>
-
-      <div className="collapse navbar-collapse" id="navbarTogglerDemo02">
-        <ul className="navbar-nav mr-auto mt-2 mt-lg-0">
-          <li className="nav-item">
-            <Link to={routes.feed()} className="nav-link" href={routes.feed()}>
-              <FormattedMessage {...messages.feed} />
+  <Wrapper id={HEADER_ID}>
+    <div className="container">
+      <div className="d-flex align-items-center justify-content-between">
+        {!props.isMenuVisible && (
+          <Logo className="d-flex align-items-center">
+            <Icon
+              onClick={props.showMenu}
+              className="mr-3 d-flex d-md-none"
+              icon={!props.isMenuVisible ? headerNavigationIcon : closeIcon}
+              noMargin
+            />
+
+            <Link to={homeRoute} href={homeRoute}>
+              <img src={img} alt="logo" />
             </Link>
-          </li>
+          </Logo>
+        )}
 
-          <li className="nav-item">
-            <Link
-              to={routes.questions()}
-              className="nav-link"
-              href={routes.questions()}
+        <Section isMenuVisible={props.isMenuVisible}>
+          {!props.isMenuVisible && [
+            <Input
+              key="1"
+              className="d-none d-lg-flex"
+              type="text"
+              placeholder={props.intl.formatMessage({ id: messages.search.id })}
+              isSearchable
+            />,
+
+            <Button
+              key="2"
+              className="d-flex justify-content-center align-items-center d-lg-none"
+              type="red"
```

Comment:
Similar to the above comment. It is better to use a role for names instead of actual color.